query-data/metrics/overview.mdx

```diff
@@ -6,12 +6,20 @@ sidebarTitle: Overview
 
 import Prerequisites from "/snippets/standard-prerequisites.mdx"
 
-Axiom supports ingesting, querying, and visualizing OpenTelemetry (OTel) metrics alongside logs and traces. This section explains how to work with OTel metrics in Axiom.
+Axiom’s dedicated MetricsDB provides a purpose-built metrics database that handles high-cardinality time-series data without the cost penalties and performance degradation common in traditional metrics systems. This section explains how to work with OpenTelemetry (OTel) metrics in Axiom.
 
 <Info>
 Support for OTel metrics is currently in private preview. For more information, see [Feature states](/platform-overview/roadmap#feature-states).
 </Info>
 
+## What makes MetricsDB different
+
+MetricsDB is engineered from the ground up to embrace dimensional complexity:
+
+- **High-cardinality as a design principle**: Store metrics with high-cardinality dimensional tags: customer IDs, trace IDs, pod names, user agents, and other highly variable dimensions. Where other metrics databases penalize you with higher costs or degraded performance, MetricsDB treats high cardinality as a core capability.
```

Comment:
Even if we don't suffer as much as other systems from high cardinality we should definitely not suggest storing `customer IDs` or `trace IDs` as tags, that is still an anti-pattern that will not make for a good experience either for us or the customers.

I would prefer not give any examples here as users tend to do strange things and giving an example implies we explicitly support that no matter how silly a user goes.